stratum/hal/bin/tdi/es2k/es2k_main.cc

```diff
@@ -57,6 +59,13 @@ static void InitCommandLineFlags() {
   // Chassis config file
   SetDefault("chassis_config_file", DEFAULT_CONFIG_DIR "es2k_port_config.pb.txt");
 
+  /* Always create a new file for pipeline_cfg.pb.txt for E2100.
```

Comment:
Is the behavior changing between different targets? (we only create a new file for E2100 target?)